itables/datatables_template.html

```diff
@@ -1,46 +1,62 @@
 <table id="table_id"><thead><tr><th>A</th></tr></thead></table>
-<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.11.3/css/jquery.dataTables.min.css">
-<style> table td {
-    text-overflow: ellipsis;
-    overflow: hidden;
-} </style>
-<style> table th {
-    text-overflow: ellipsis;
-    overflow: hidden;
-} </style>
+<style>
+    table td {
+        text-overflow: ellipsis;
+        overflow: hidden;
+    }
+</style>
+<style>
+    table th {
+        text-overflow: ellipsis;
+        overflow: hidden;
+    }
+</style>
 <script type="module">
-    // Define the table data
-    const data = [];
+    // This function adds the datatables CSS to the HTML header
+    const loadStyle = (url) => {
+        let style;
+        const styles = Array.from(document.head.querySelectorAll('link'));
+        const existingStyle = styles.find((style) => style.href === url);
+        if (!existingStyle) {
+            style = document.createElement('link');
+            style.rel = 'stylesheet';
+            style.type = 'text/css';
+            style.href = url;
+            document.head.appendChild(style);
+        }
+    };
 
-    if (typeof require === 'undefined') {
-        // TODO: This should become the default (use a simple import)
-        // when the ESM version works independently of whether
-        // require.js is there or not, see
-        // https://datatables.net/forums/discussion/69066/esm-es6-module-support?
-        const {default: $} = await import("https://esm.sh/jquery@3.5.0");
-        const {default: initDataTables} = await import("https://esm.sh/datatables.net@1.11.3?deps=jquery@3.5.0");
+    // Import jQuery and datatables
+    let $;
+    let dt;
+    try {
+        const { default: static_$ } = await import("/static/itables/jquery/src/jquery.js");
```

Comment:
@fwouts , is there any way to avoid creating the variable `static_$` and assign the import directly to `$`?

Reply:
Also when the file is not found some errors appear in the log e.g.
![image](https://user-images.githubusercontent.com/29915202/174326514-6bc55f77-606a-4bc9-9806-2633262374e9.png)

Is there any way to avoid printing these errors (since anyway we're going to import the libraries from the web in that case)?

Reply:
> @fwouts , is there any way to avoid creating the variable `static_$` and assign the import directly to `$`?

Yes!
```js
$ = (await import("/static/itables/jquery/src/jquery.js")).default;
```

Reply:
> Also when the file is not found some errors appear in the log e.g. ![image](https://user-images.githubusercontent.com/29915202/174326514-6bc55f77-606a-4bc9-9806-2633262374e9.png)
> 
> Is there any way to avoid printing these errors (since anyway we're going to import the libraries from the web in that case)?

It's possible that overriding `window.onerror` would help: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror. However I think at least some browsers will still show the error in the logs :(